testSuite/__main__.py

```diff
@@ -1,107 +1,113 @@
 import os
+import sys
 import io
-import traceback
-import shutil
 import unittest
 import subprocess
 
-from .util import *
+from .util import assertFilesEqual, deleteFiles
 
 TARGET = ''
-NEEDS  = [ ]

```

Comment:
please don't change the style, these kind of changes is pretty hard to review on without tools like gumtree. This commit won't be accepted **for now***.